student.rb

```diff
@@ -4,9 +4,9 @@
 class Student < Person
   attr_accessor :classroom
 
-  def initialize(classroom, age, name = 'Unknown', parent_permission = 'true')
+  def initialize(age, name = 'Unknown', parent_permission: 'true')
     super(age, name, parent_permission)
-    @classroom = classroom
+    # @classroom = classroom
```

Comment:
- [ ] Do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. 👍🏻